src/drt/src/dr/FlexDR.h

```diff
@@ -687,11 +687,11 @@ class FlexDRWorker
   void initTrackCoords(
       std::map<frCoord, std::map<frLayerNum, frTrackPattern*>>& xMap,
       std::map<frCoord, std::map<frLayerNum, frTrackPattern*>>& yMap);
-  void initTrackCoords_route(
+  void inittrack_coords_route(
```

Comment:
This and the other camel case examples are an oversight. I'm still working on some renaming and making Doxygen documentation. I used the VScode feature to change very instance of some variable names that were in camelCase to snake_case, and it changed some function names as the name was contained there. This will be fixed.